### PR TITLE
Fix setup.cfg name 

### PR DESCRIPTION
**Pull request by C-Achard**
Fixes warning when building from setuptools :

        ********************************************************************************
        Usage of dash-separated 'napari-cellseg3d' will not be supported in future
        versions. Please use the underscore name 'napari_cellseg3d' instead.
        By 2024-Sep-26, you need to update your project and remove deprecated calls
        or your builds will no longer be supported.
        See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

### CODE REVIEW

**Comment by C-Achard** [comment]
@MMathisLab Just to confirm, this will not affect the project name on PyPi, correct ? 
I tried on test-pypi (https://test.pypi.org/project/napari-cellseg3d/) manually with twine, it seems to upload and then install fine, I just want to make sure switching to napari_cellseg3d instead of napari-cellseg3d will not try to upload to a different PyPi project

**Comment by MMathisLab** [comment]
this I don't know, but suspect fine -- if you can push it then I think its okay

File: README.md [hunk 1]
```diff
@@ -39,12 +39,15 @@ Then go into Plugins > napari-cellseg3d, and choose which tool to use.
 
 ## News
 
-**New version : v0.1.2**
+**New version : v0.1.3**
 
-- Fixed manifest issue for PyPi
+- Small API changes for training/inference from a script
+- Changed project name to "napari_cellseg3d" to avoid setuptools deprecation
```
Comment (MMathisLab) [inline README.md:45]: a name change should not be a patch, this should be at min, a minor change, so bump to 0.2

**Review by MMathisLab — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (C-Achard) [reply README.md:45]: Okay, perhaps we can discuss it in #71 ?

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**